README.md

```diff
@@ -1,17 +1,16 @@
-# arcgis-maps-sdk-unity-samples
+# Esri CloudXR Demo
 
 ![image](arcgis-maps-sdk-unity-samples.png)
 
-Here is a Unity project containing a set of samples showing you how to accomplish various things using the combined features of Unity and the ArcGIS Maps SDK for Unity. The `main` branch is configured to work with our most recent release (1.2.0) if you want to use the sample repo with an older release check out the corresponding tag of the sample repo, `git checkout 1.0.0` for the sample repo that worked with our 1.0.0 release.
+Here is a VR demo developed by ESRI within the Unity Engine. It is built off the [ArcGIS SDK Unity VRSample](https://github.com/Esri/arcgis-maps-sdk-unity-samples/tree/main/samples_project/Assets/SampleViewer/Samples/VRSample), incorporating more intractability and diverse features in order to utilize the demo in tandem with [Nvidia’s CloudXR Application](https://developer.nvidia.com/cloudxr-sdk). The main branch is configured to work with our ArcGIS plugin version 1.3.0. If you would like to build the application to a VR headset you would need to switch to the URPBuild branch since Androids do not support the HD Rendering Pipeline we have implemented in the main branch.
```

Comment:
```suggestion
Here is a VR demo developed by ESRI within the Unity Engine. It is built off the [ArcGIS SDK Unity VRSample](https://github.com/Esri/arcgis-maps-sdk-unity-samples/tree/main/samples_project/Assets/SampleViewer/Samples/VRSample), incorporating more interaction and diverse features in order to utilize the demo in tandem with [Nvidia’s CloudXR Application](https://developer.nvidia.com/cloudxr-sdk). The main branch is configured to work with our ArcGIS plugin version 1.3.0. If you would like to build the application to a VR headset you would need to switch to the URPBuild branch since Android does not support the HD Rendering Pipeline we use in the main branch.
```